calico_kubernetes/calico_kubernetes.py

```diff
@@ -741,3 +741,23 @@ def _label_to_tag(self, label_key, label_value):
         elif mode == 'status':
             logger.info('Executing Calico pod-status hook')
             NetworkPlugin().status(namespace, pod_name, docker_id)
+
+
+if __name__ == '__main__':
+    # Configure logging
+    configure_logger(logger, LOG_LEVEL, True)
+    configure_logger(pycalico_logger, LOG_LEVEL, False)
+
+    # Try to run the plugin, logging out any BaseExceptions raised.
+    logger.info("Begin Calico network plugin execution")
+    try:
+        main()
+    except SystemExit:
+        # If a SystemExit is thrown, we've already handled the error and have called sys.exit()
+        # No need to produce a duplicate exception message, so exit without logging.
+        os._exit(1)
```

Comment:
>Why catch SystemExit and then os._exit? Why not just os._exit inside the call?

This allows us to continue to use sys.exit() in our code, which is a lot easier to UT using `self.assertRaises(some_func, SystemExit)`.